### DISCUSSION TITLE
如何在mirai组件中使用 fix-protocol-version

### DISCUSSION THREAD

**Discussion by ikaros143** [simple-robot/Discussions#108] [Q&A] [answered]
### 涉及的编程语言

Java

### 疑问描述

现在用simbot已经彻底登不上去了，只能用mrai了，直接打成jar包放进plugins里面好像没有效果，也没任何提示，是要修改什么配置吗？既然用mrai登录了，我打包之前要把账号的配置文件给删掉吗？

**Comment by ForteScarlet** [comment, marked as answer]
simbot的mirai组件并不是mirai插件工程，自然不能打包成 `mirai plugin`。

虽然理论上mirai能登上去就可以用simbot登，不过既然你选择直接使用mirai，那么就没simbot什么事儿了。

**Reply by ikaros143** [reply]
今天更新了fix-protocol-version插件后才能登上去，但是我并不知道这个插件要怎么用在simbot里面，即使用aoki把登录成功的信息复制过来也没有登陆成功。

**Reply by ForteScarlet** [reply]
`fix-protocol-version` 的README中有针对 `mirai-core` 的[使用方法](https://github.com/cssxsh/fix-protocol-version#mirai-core-%E4%BD%BF%E7%94%A8%E6%96%B9%E6%B3%95)，看样子在main方法开始的时候调用其指定方法即可。

**Reply by ikaros143** [reply]
已经解决了，非常感谢！！

**Reply by TotoroKingdom** [reply]
怎么解决的
